### PR TITLE
Fix/extract opcode

### PR DESCRIPTION
**Pull request by vuvoth**
## Proposed Changes

+ Parameter in extract* opcode should be bigint

## Test plan 
+ Unit test.

### CODE REVIEW

**Review by sczembor — APPROVED** [review]
Should we update the changeling in this case?

**Comment by vuvoth** [comment]
> Should we update the changeling in this case?

Yes, I forgot it. Thanks

File: packages/runtime/test/src/interpreter/opcode-list.ts [hunk 1]
```diff
@@ -5700,7 +5716,16 @@ describe("Teal Opcodes", function () {
...
 			op.execute(stack);
 
-			const expected = bigEndianBytesToBigInt(new Uint8Array([3, 4]));
+			let expected = bigEndianBytesToBigInt(new Uint8Array([3, 4]));
+			assert.equal(stack.pop(), expected);
+
+			// long byte
+			stack.push(longByte);
+			stack.push(300n);
+
```
Comment (rmeena840) [inline packages/runtime/test/src/interpreter/opcode-list.ts:5725]: There is a lot of empty lines I see. You should remove them wherever not needed.

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviewed. Left comments.

File: packages/runtime/test/src/interpreter/opcode-list.ts (hunk 1 above)
Comment (vuvoth) [reply packages/runtime/test/src/interpreter/opcode-list.ts:5725]: Done

**Review by rmeena840 — APPROVED** [review]

**Pull request merged**